corehq/motech/openmrs/const.py

```diff
@@ -117,9 +118,7 @@
     'endDate': OPENMRS_DATA_TYPE_DATETIME,
 }
 OPENMRS_PROPERTIES = dict(chain(
-    # pylint: disable=W1654
```

Comment:
@kaapstorm I see what you tried to do here, but unfortunately modernize doesn't respect these flags.  I'm changing the code to use ```six``` to keep noise from modernize low.